model/src/main/kotlin/VcsInfo.kt

```diff
@@ -51,7 +52,14 @@ data class VcsInfo(
          * example, for Git only this subdirectory of the repository should be cloned, or for Git Repo it is
          * interpreted as the path to the manifest file.
          */
-        val path: String = ""
+        val path: String = "",
+
+        /**
+         * The VCS-specific revision resolved during downloading from the VCS. In contrast to [revision] this must not
+         * contain symbolic names like branches or tags.
+         */
+        @JsonInclude(JsonInclude.Include.NON_NULL)
+        val resolvedRevision: String? = null
```

Comment:
Should we sort this in directly below `revision`?

Reply:
Done.

Reply:
No, actually I like that Kotlin feature and the parameter order makes more sense this way.